wsi/physicaldisplay.h

```diff
@@ -63,6 +63,10 @@ class PhysicalDisplay : public NativeDisplay, public DisplayPlaneHandler {
   bool SetActiveConfig(uint32_t config) override;
   bool GetActiveConfig(uint32_t *config) override;
 
+  bool SetResolution(const HwcRect<int32_t>&) override;
+  bool IsCustomResolution();
```

Comment:
Do we need this ?

Reply:
Are you commenting on 'override' keyword?

Reply:
No this API bool IsCustomResolution();

Reply:
Yes, for cleaner implementation, this is required